fedora_distro_aliases/__init__.py

```diff
@@ -6,33 +6,42 @@
 
 import requests
 from munch import Munch
+from fedora_distro_aliases.cache import Cache
 
 
 def bodhi_active_releases():
     """
     Return all active releases from Bodhi
     https://bodhi.fedoraproject.org
     """
-    bodhi_url =  "https://bodhi.fedoraproject.org/releases/"
+    bodhi_url = "https://bodhi.fedoraproject.org/releases/"
     releases = []
     states = ["current", "pending", "frozen"]
     for state in states:
         url = "{0}?state={1}".format(bodhi_url, state)
         response = requests.get(url)
         response.raise_for_status()
         releases.extend(response.json()["releases"])
-    return [Munch(x) for x in releases]
+    return releases
 
 
-def get_distro_aliases():
+def get_distro_aliases(cache=None):
     """
     Define distribution aliases like `fedora-stable`, `fedora-branched`,
     `epel-all`, and more.
     """
-    releases = bodhi_active_releases()
-    epel = []
-
-    distros = [Distro.from_bodhi_release(x) for x in releases if x.name != "ELN"]
+    if cache is True:
+        cache = Cache()
+    try:
+        releases = bodhi_active_releases()
+        if cache:
+            cache.save(releases)
+    except requests.exceptions.RequestException as ex:
+        if not cache:
```

Comment:
~~Do you override `__bool__` so that it returns `False` when the cache is empty, i.e., _I'm caching, created a valid cache, but failed to download the releases for the first time_?~~

Never mind, it raises `BadCache` in such event…